app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginViewModel.java

```diff
@@ -96,4 +110,25 @@ protected void onCleared() {
         super.onCleared();
         compositeDisposable.dispose();
     }
+
+     public String getEmail() {
+        String email;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
+            email = EncryptionUtils.decryptString(encryption.getContext(), sharedPreferenceModel.getString(Constants.PREF_USER_EMAIL, null));
+        } else {
+            email = sharedPreferenceModel.getString(Constants.PREF_USER_EMAIL, null);
+        }
+        return email;
+    }
+
+    public String getUserpassword() {
+        String password;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
+            password = EncryptionUtils.decryptString(encryption.getContext(), sharedPreferenceModel.getString(Constants.PREF_USER_PASSWORD, null));
+        } else {
+            password = sharedPreferenceModel.getString(Constants.PREF_USER_PASSWORD, null);
+        }
+        return password;
+    }
```

Comment:
What i would suggest doing is create a repository called `EncryptionRepository`. You can move these two methods there and just call it from the vm if needed. This would mean that the VM is like a clean API and the Model does all the heavy lifting.